web/src/pages/api/v1/precheck/[app_id].ts

```diff
@@ -110,12 +111,9 @@ const createActionQuery = gql`
 `;
 
 const schema = yup.object({
-  action: yup.string().strict(),
+  action: yup.string().strict().required("This attribute is required."),
```

Comment:
we should require that it includes either action or external nullifier to avoid breaking any existing usage